src/routes/jobs/+page.svelte

```diff
@@ -173,18 +186,121 @@
         'Erfahrungen Docker / Kubernetes / Gitlab CI',
       ],
     },
-    // {
-    //   title: '(Dev)Ops Engineer',
-    //   content:
-    //     'Als (Dev)Ops betreust du unsere Projekte im Unterhalt. Du reagierst auf Meldungen in unserem Monitoringsystem und verbesserst dieses laufend. Du hilfst im Team Infrastrukturen in unserer Projekte aufzubauen und diese zu automatisieren. Dein Ziel ist eine stabile Umgebung für die Entwicklung und den Kunden zu liefern.',
-    //
-    //   image: { src: 'img/jobs/cloud_hosting.svg', alt: 'Dev/Ops Engineer' },
-    //   bulletPoints: [
-    //     'Erfahrungen Docker / Kubernetes / Gitlab CI',
-    //     'Erfahrungen mit Google Cloud / Azure von Vorteil',
-    //     'Erfahrungen mit Cypress / E2E Testing von Vorteil',
-    //   ],
-    // },
+    {
+      formReference: '(Dev)Ops Engineer',
+      title: '(Dev)Ops Engineer',
+      content:
+        'Als (Dev)Ops betreust du unsere Projekte im Unterhalt. Du reagierst auf Meldungen in unserem Monitoringsystem und verbesserst dieses laufend. Du hilfst im Team Infrastrukturen in unserer Projekte aufzubauen und diese zu automatisieren. Dein Ziel ist eine stabile Umgebung für die Entwicklung und den Kunden zu liefern.',
+
+      image: { src: 'img/jobs/cloud_hosting.svg', alt: 'Dev/Ops Engineer' },
+      bulletPoints: [
+        'Erfahrungen Docker / Kubernetes / Gitlab CI',
+        'Erfahrungen mit Google Cloud / Azure von Vorteil',
+        'Erfahrungen mit Cypress / E2E Testing von Vorteil',
+      ],
+    },
+  ]
+
+  let companyInfo: BlockContent[] = [
+    {
+      title: 'Was wir dir bieten',
```

Comment:
hast du das dupliziert? wäre es nicht besser das auszulagern und wiederzuverwenden?

Reply:
Ich hab das verschoben damits am selben ort ist wie der rest. Hatte mir aber eh überlegt obs nicht eventuell klug wär solche sachen in eigene files auszulagern damit man diese beschreibungen allgemein einfacher editieren/wiederverwenden kann, die pages in denen beschreibungen mit drin sind werden meiner meinung nach schnell ziemlich unübersichtlich